test/fstapp_mnesia_SUITE.erl

```diff
@@ -0,0 +1,25 @@
+-module(fstapp_mnesia_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+
+-include_lib("stdlib/include/qlc.hrl").
+
+-compile(export_all).
+
+init_per_testcase(_, Config) ->
+
+	ok = application:set_env(fstapp, callback_module, fstapp_mnesia_handler),
+	{ok, fstapp_mnesia_handler} = application:get_env(fstapp, callback_module),
+	_ = application:ensure_all_started(fstapp),
+	Config.
+
+end_per_testcase(_, _Config) ->
+	application:stop(fstapp).
+
+all() -> [mnesia_test].
+
+mnesia_test(_Config) ->
+
+	timer:sleep(10000),
```

Comment:
don't stop the system 10 seconds, if you set the frequency to 200 you can sleep only one second